app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,70 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if !logged_in?
+      redirect to '/login'
+    end
+    @tweets = Tweet.all
+    erb :'tweets/tweets'
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    @tweet = Tweet.create(params)
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.